### PR TITLE
Use io dispatcher

### PR DESCRIPTION
**Pull request by Horneth**
Makes sure the I/O operations run on the I/O dispatcher

### CODE REVIEW

File: services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala [hunk 1]
```diff
@@ -58,7 +58,7 @@ class WriteMetadataActor(batchRate: Int, flushRate: FiniteDuration)
       addMetadataEvents(e.toVector) onComplete {
         case Success(_) => self ! DbWriteComplete
         case Failure(regerts) =>
-          log.error("Failed to properly flush metadata to database", regerts)
+          log.error(regerts, "Failed to properly flush metadata to database")
```
Comment (mcovarr) [inline services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala]: The spelling is regerttable.

**Comment by mcovarr** [comment]
:+1:

File: services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala (hunk 1 above)
Comment (geoffjentry) [inline services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala]: Someone has never seen the snickers commercial

Comment (geoffjentry) [inline services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala]: https://www.quora.com/What-does-the-word-regerts-mean

Can you tell who did it? :)

**Comment by geoffjentry** [comment]
👍

File: services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala (hunk 1 above)
Comment (mcovarr) [inline services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala]: 🍫

File: engine/src/main/scala/cromwell/engine/io/IoActor.scala [hunk 2]
```diff
@@ -74,7 +75,7 @@ final class IoActor(queueSize: Int, throttle: Option[Throttle])(implicit val mat
       .via(flow)
   } getOrElse flow
   
-  override protected lazy val streamSource = source.via(throttledFlow).withAttributes(ActorAttributes.dispatcher(Dispatcher.IoDispatcher))
+  override protected lazy val streamSource = source.via(throttledFlow)
```
Comment (cjllanwarne) [inline engine/src/main/scala/cromwell/engine/io/IoActor.scala]: I didn't see why this one is no longer using the dispatcher?

Comment (Horneth) [inline engine/src/main/scala/cromwell/engine/io/IoActor.scala]: Good point I'll put it back

**Pull request merged**